run-k8s-external-storage-e2e.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/sh
+#
+# Download the Kubernetes test suite, extract it and run the external-storage suite.
+#
+# Requirements:
+# - KUBE_VERSION needs to be set in the environment (format: "v1.18.5")
+
+# exit on failure
+set -e
+
+[ -n "${KUBE_VERSION}" ] || { echo "KUBE_VERSION not set" ; exit 1 ; }
+
+# download and extract the tests
+curl -LO "https://storage.googleapis.com/kubernetes-release/release/${KUBE_VERSION}/kubernetes-test-linux-amd64.tar.gz"
+tar xzf kubernetes-test-linux-amd64.tar.gz kubernetes/test/bin/ginkgo kubernetes/test/bin/e2e.test
+
+# generate the rbd-driver.yaml that describes the test features
+cat > rbd-driver.yaml << EOF
+ShortName: cephcsi-rbd-test
+StorageClass:
+  FromExistingClassName: csi-rbd-sc
+  #FromFile: examples/rbd/storageclass.yaml
+
+SnapshotClass:
+  # Must be set to enable snapshotting tests
+  FromName: true
+
+DriverInfo:
+  # Internal name of the driver, this is used as a display name in the test case and test objects
+  Name: test.rbd.csi.ceph.com
+
+  # The range of disk size supported by this driver
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 16Ti
+
+  # Map of strings for supported FS types
+  SupportedFsType:
+    ext4: {}
+    xfs: {}
+
+  # Map of strings for supported mount options
+  SupportedMountOption:
+    rw: {}
+
+  # Map of strings for required mount options
+  RequiredMountOption:
+    rw: {}
+
+  # Optional list of access modes required for provisiong. Default is RWO
+  # RequiredAcccessModes:
+
+  # Map that represents the capabilities the driver supports
+  Capabilities:
+    # Data is persistest accross pod restarts
+    persistence: true
+
+    # Volume ownership via fsGroup
+    fsGroup: false
+
+    # Raw block mode
+    block: true
+
+    # Exec a file in the volume
+    exec: true
+
+    # Support for volume limits
+    volumeLimits: false
+
+    # Support for volume expansion in controllers
+    controllerExpansion: false
+
+    # Support for volume expansion in nodes
+    nodeExpansion: false
+
+    # Support volume that an run on single node only (like hostpath)
+    singleNodeVolume: false
+
+    # Support ReadWriteMany access modes
+    RWX: false
```

Comment:
we support RWX for block mode do we have an option to set it?

Reply:
Yes, but enabling RWX also tests filesystem mode as RWX and that fails. I have not found an option to only enable RWX for block.

Reply:
There is also an other issue, see kubernetes/kubernetes#94882 for the bug against the test-case.